### PR TITLE
Syntax error in Travis config

### PR DESCRIPTION
**Pull request by szepeviktor**
`grep: Unmatched ( or \(`

### CODE REVIEW

**Comment by szepeviktor** [comment]
We **have** `isinstance(`

File: .travis.yml [hunk 1]
```diff
@@ -229,7 +229,7 @@ script:
 
   # home-made bad code detectors
   - if [ "$MODE" == 'SCA' ]; then egrep -r '^ *class [^\(]+(\(\))?:' . && exit 1 || true; else echo skip; fi  # old-style class detector
-  - if [ "$MODE" == 'SCA' ]; then grep -r 'isinstance\(' . && exit 1 || true; else echo skip; fi  # very non-pythonic
+  - if [ "$MODE" == 'SCA' ]; then grep -r 'isinstance(' . && exit 1 || true; else echo skip; fi  # very non-pythonic
```
Comment (ppolewicz) [inline .travis.yml]: you need to escape it somehow so that `.travis.yml` is not matched

Comment (szepeviktor) [inline .travis.yml]: Done.

Comment (szepeviktor) [inline .travis.yml]: Could you replace `isinstance()` instances in the Python code?

**Comment by szepeviktor** [comment]
@ppolewicz Is it ready to merge?

**Comment by ppolewicz** [comment]
@szepeviktor it fails tests, because there are some `isinstance` used in the code, so we cannot merge it like that.

**Comment by szepeviktor** [comment]
Oh yes :(
https://github.com/Backblaze/B2_Command_Line_Tool/search?utf8=%E2%9C%93&q=isinstance&type=

**Comment by bwbeach** [comment]
Is this still relevant?  Will close soon if nobody speaks up.

**Comment by szepeviktor** [comment]
> @szepeviktor it fails tests, because there are some isinstance used in the code, so we cannot merge it like that.

As @ppolewicz said the code need some cleanup:
https://github.com/Backblaze/B2_Command_Line_Tool/search?utf8=%E2%9C%93&q=isinstance&type=

**Comment by mlech-reef** [comment]
This PR will be obsolete after #650

**Comment by szepeviktor** [comment]
Oh yes.

**Pull request closed without merging**